<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultModify</class>
 <widget class="QDialog" name="ResultModify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>结果修改</string>
  </property>
  <widget class="QLabel" name="lbRMNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRMOldResult">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRMTestDate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>测试时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRMNewResult">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>修改结果</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRMNumber">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRMTestDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRMOldResult">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRMNewResult">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRMOK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
